pkg/osquery/extension.go

```diff
@@ -786,6 +797,19 @@ func (e *Extension) writeResultsWithReenroll(ctx context.Context, results []dist
 }
 
 func getEnrollDetails(client Querier) (service.EnrollmentDetails, error) {
+	var details service.EnrollmentDetails
+
+	if os.Getenv("LAUNCHER_DEBUG_ENROLL_DETAILS_ERROR") == "true" {
```

Comment:
what is this going to be used for? please leave comment to explain

Reply:
Heh. You both flagged this. I've added a comment